### ISSUE TITLE
write_kafka plugin: rd_kafka_set_logger() is deprecated in librdkafka-0.9.1

### ISSUE THREAD

**Issue by octo** [collectd/collectd#2029]
*   Version of collectd: *master*
*   Operating system / distribution: FreeBSD 10

## Expected behavior

*n/a*

## Actual behavior

```
write_kafka.c: In function 'kafka_handle':
write_kafka.c:119:6: error: 'rd_kafka_set_logger' is deprecated (declared at /usr/local/include/librdkafka/rdkafka.h:2400) [-Werror=deprecated-declarations]
      rd_kafka_set_logger(ctx->kafka, kafka_log);
      ^
```

**Comment by octo** [comment]
It appears the new function is called `rd_kafka_conf_set_log_cb()` and has the same signature.

**Comment by octo** [comment]
This was deprecated in edenhill/librdkafka@8c4cf4e4151d0b1cf7136052bf98f834e61ee670, which was first included in version 0.9.0.
